packages/shared/src/components/modals/NewSourceModal.tsx

```diff
@@ -195,13 +196,8 @@ export default function NewSourceModal(props: ModalProps): ReactElement {
             autoFocus
             aria-describedby={scrapeError && 'new-source-field-desc'}
             valueChanged={onUrlChanged}
-            fieldType="secondary"
-            rightButtonProps={{
-              disabled: !enableSubmission,
-              type: 'submit',
-              'aria-label': 'Search feeds',
-              buttonSize: 'small',
-            }}
+            fieldType="primary"
```

Comment:
Is it part of the new design to turn this into a primary?

Reply:
yeap https://www.sketch.com/s/2867ec63-0966-4647-b1d7-b6fe696ba251/a/m1913YJ